src/content-tags-drawer/data/apiHooks.jsx

```diff
@@ -147,7 +147,7 @@ export const useContentTaxonomyTagsUpdater = (contentId) => {
         contentPattern = contentId.replace(/\+type@.*$/, '*');
       }
       queryClient.invalidateQueries({ queryKey: ['contentTagsCount', contentPattern] });
-      if (contentId.includes('lb:')) {
+      if (contentId.startsWith('lb:') || contentId.startsWith('lib-collection:')) {
         // Obtain library id from contentId
         const libraryId = ['lib', ...contentId.split(':').slice(1, 3)].join(':');
```

Comment:
Nit: would be nicer to use [`getLibraryId`](https://github.com/openedx/frontend-app-authoring/blob/1ee3229104d96d47235fe89a88317ab6c0cb41f7/src/generic/key-utils.ts#L21) though you'd have to update it (and its tests) to work with `lib-collection:` keys as well.

Reply:
Updated: https://github.com/openedx/frontend-app-authoring/pull/1379/commits/bdc1fd89d9a2c620968b33a9e96c2245fb473145